cg/cli/clean.py

```diff
@@ -207,6 +207,7 @@ def remove_invalid_flow_cell_directories(context: CGConfig, failed_only: bool, d
     status_db: Store = context.status_db
     demux_api: DemultiplexingAPI = context.demultiplex_api
     housekeeper_api: HousekeeperAPI = context.housekeeper_api
+    sample_sheets_dir: str = context.clean.flow_cells.sample_sheets
```

Comment:
I am a bit confused by the variable naming.  `context.clean.flow_cells.sample_sheets` feels like it should be a list of sample_sheets like List[Path] or List[str]. But, you actually get a directory back which is a str and not a path so it should actually be dir_name?